juju/model.py

```diff
@@ -930,9 +966,7 @@ def debug_log(
 
         """
         if to:
-            placement = [
-                client.Placement(**p) for p in to
-            ]
+            placement = parse_placement(to)
```

Comment:
You mean the one in Application.add_unit()? AFAICT that's still needed, what am I missing?